expat/tests/fuzz/parse_fuzzer.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "expat.h"
+
+#include <functional>
```

Comment:
Are we using anything from `<functional>`?

Reply:
Not sure

Reply:
Let's drop it and see if it still compiles :smile: